src/audio/Kconfig

```diff
@@ -467,6 +467,12 @@ config PCM_CONVERTER_FORMAT_S24LE
 	help
 	  Support 24 bit processing data format with sign and in little endian format
 
+config PCM_CONVERTER_FORMAT_S24_3LE
+	bool "Support S24_3LE"
+	default y
```

Comment:
can we make this default n to keep the code size small and have IPC4 mode select it.

Reply:
sure